### DISCUSSION TITLE
Which is the best vision language model?

### DISCUSSION THREAD

**Discussion by ShreyasG482** [community/community#147530] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I want a VLN which is the best for OCR

**Comment by Sharath1036** [comment, marked as answer]
GPT-4o is the best vision model and returns the accurate ocr for any quality of image. It's that powerful.
An open source alternative would be Qwen VL Chat from HuggingFace. Grayscaling images would be better for Qwen. 
There's a Deepseek vision model in the market too, I'll check and update

**Reply by ShreyasG482** [reply]
Thankss

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @ShreyasG482!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

**Reply by Singen54** [reply]
هلا
